.github/workflows/publish.yaml

```diff
@@ -0,0 +1,18 @@
+name: Publish Package
+
+on:
+  release:
+    types: [created]
```

Comment:
How does this get invoked? Do we need to run `npm version` somewhere?

Reply:
As far as I understand, when a new release gets created on https://github.com/vercel/serve-handler/releases

So, we'd continue to run  e.g. `release` to create a new version + update package.json + prepare the release. And once the release gets published/created, that action would run.

Reply:
I see, so there will still be a manual process and committing/pushing to main directly.

In the future, we might want to automate that part too 👍

Reply:
We’re using https://github.com/changesets/changesets in [blockprotocol/blockprotocol](https://github.com/blockprotocol/blockprotocol), works pretty well for managing changelogs & releases on GitHub!